### ISSUE TITLE
Invalid PropType warning on 1.4.0

### ISSUE THREAD

**Issue by andrewmclagan** [techniq/mui-downshift#76]
```javascript
Warning: Invalid arguments supplied to oneOf, expected an array, got 3 arguments. A common mistake is to write oneOf(x, y, z) instead of oneOf([x, y, z]).
```

Seeming to come from https://github.com/techniq/mui-downshift/blob/781fbb2b75614fd331ce7bef894d7e5a0a1f96da/src/index.js#L110 that was updated with this version.

PR coming

**Comment by andrewmclagan** [comment]
see PR https://github.com/techniq/mui-downshift/pull/77

**Comment by cvanem** [comment]
@andrewmclagan Thanks for catching that.  I don't use PropTypes much :)

**Comment by techniq** [comment]
Fixed in [1.4.1](https://github.com/techniq/mui-downshift/releases/tag/v1.4.1).  Thanks for the PR.
